### PR TITLE
mgmtsystem_review migration to odoo 9 progress

### PR DESCRIPTION
**Pull request by naousse**

### CODE REVIEW

**Comment by naousse** [comment]
@dreispt 
Please Daniel you can review this too

**Comment by dreispt** [comment]
LGTM code review no test :+1:

File: mgmtsystem_review/README.rst [hunk 1]
```diff
@@ -42,6 +42,7 @@ Contributors
 
 * Maxime Chambreuil <maxime.chambreuil@savoirfairelinux.com>
 * Loïc Faure-Lacroix <loic.lacroix@savoirfairelinux.com>
+* Gervais Naoussi <gervaisnaoussi@gmail.com>
```
Comment (max3903) [inline mgmtsystem_review/README.rst:45]: Please add `Luk Vermeylen <luk@allmas-it.be>`

File: mgmtsystem_review/__init__.py [hunk 2]
```diff
@@ -1,6 +1,3 @@
-# -*- encoding: utf-8 -*-
+# -*- coding: utf-8 -*-
 
-from . import (
-    models,
-    report,
-)
+from . import models, tests
```
Comment (dreispt) [inline mgmtsystem_review/__init__.py]: don't import tests
they are automatically discovered by unittest

**Comment by dreispt** [comment]
+1 for the tests

**Comment by max3903** [comment]
@naousse Please:
- [x] Rebase
- [x] Add `Luk Vermeylen <luk@allmas-it.be>` as a contributor

**Comment by naousse** [comment]
Luk Vermeylen luk@allmas-it.be  is allready added

**Comment by max3903** [comment]
@naousse 
- [x] `Reviews` menu item should be under the `Audits` one
- [x] `Policy` fields can be html text.
- [ ] There is no `Add item` in the survey answers section. I can't select survey answers and link them to the review.
- [x] `Company` field in the review line form and tree views should be displayed only to the multi-company group.

In the review report:
- [x] `Confidentially` --> `Confidentiality`

**Comment by naousse** [comment]
@max3903 
Please i don't understand why we don't have the add item button.
I find that everithing is correct with our source code
Please can you check it too

**Comment by naousse** [comment]
@dreispt 
Please, can you check this problem 
In mgmtsystem_review model
I have this
response_ids = fields.Many2many(
        'survey.user_input',
        'mgmtsystem_review_response_rel',
        'response_id',
        'mgmtsystem_review_id',
        'Survey Answers')
And in the view

File: mgmtsystem_review/views/mgmtsystem_review.xml [hunk 3]
```diff
@@ -135,7 +135,7 @@
                                 </group>
                                 <newline/>
                                 <separator string="Survey Answers" colspan="4"/>
-                                <field name="response_ids" colspan="4" nolabel="1" attrs="{'readonly':[('state','=','done')]}"/>
+                                <field name="response_ids" nolabel="1" attrs="{'readonly':[('state','=','done')]}"/>
```
Comment (naousse) [inline mgmtsystem_review/views/mgmtsystem_review.xml]: @dreispt 
this is how the fields is inserted in the view.
But th Add Item Button doesn't appear I don't know why

File: mgmtsystem_review/models/mgmtsystem_review.py [hunk 4]
```diff
@@ -48,7 +48,7 @@ class MgmtsystemReview(models.Model):
         'response_id',
         'mgmtsystem_review_id',
         'Survey Answers')
-    policy = fields.Text('Policy')
+    policy = fields.Html('Policy')
     changes = fields.Text('Changes')
```
Comment (max3903) [inline mgmtsystem_review/models/mgmtsystem_review.py]: @naousse `changes = fields.Html('Changes')`

and below 

`conclusion = fields.Html('Conclusion')`

**Comment by max3903** [comment]
@naousse 
- [x] `Changes...` and `Conclusion` fields can be html text.
- [x] There is no `Add item` in the survey answers section. I can't select survey answers and link them to the review.

In the review report:
- [x] Alignment is incorrect. See screenshot:
  review_report

File: mgmtsystem_review/views/mgmtsystem_review.xml (hunk 3 above)
Comment (naousse) [inline mgmtsystem_review/views/mgmtsystem_review.xml]: Fixed

**Comment by naousse** [comment]
@max3903 
I fixed the problems but, there is one now in the report. But it comes from this line that i took from last report 
`<section>
        <para>[[ repeatIn(o.response_ids,'response') ]]</para>
        <para style="Heading">[[ response.question_ids[0].single_text ]] ([[ response.question_ids[1].single_text ]])</para>
        <para leftIndent="15">Comments: [[ response.question_ids[2].comment ]]</para>
        <para leftIndent="15">Expectations: [[ response.question_ids[5].comment ]]</para>
        <para leftIndent="15">Improvements: [[ response.question_ids[3].comment ]]</para>
    </section>`
There is no questions_ids fields in survey.user_input. I can rewrite it if you explain me what we want to do there.

**Comment by max3903** [comment]
@naousse The mgmtsystem_survey module provides a customer satisfaction survey. The survey asks for the name of the customer, his comments, expectations and improvements. We want to get this information in the review to analyze the answers.

**Comment by max3903** [comment]
@naousse Here is the survey:
https://github.com/OCA/management-system/blob/7.0/mgmtsystem_survey/mgmtsystem_survey.xml

**Comment by max3903** [comment]
:+1:

**Pull request merged**